static/app/views/dashboards/editAccessSelector.tsx

```diff
@@ -79,7 +79,7 @@ function EditAccessSelector({
             ) ?? []),
           ];
     setSelectedOptions(selectedOptionsFromDashboard);
-  }, [dashboard, teamsToRender]);
+  }, [dashboard, teamsToRender, isMenuOpen]);
```

Comment:
Is this meant to be here btw? It doesn't seem like its a true dependency of the hook here

Reply:
This seems like it's necessary to cancel out a selection. Let's leave a comment so it's clear it's not an unnecessary prop!